tests/test_models.py

```diff
@@ -106,3 +106,10 @@ def test_get_message(self):
                 date(log.timestamp, "DATETIME_FORMAT"), log.content_object
             ),
         )
+
+    def test_get_for_object(self):
+        with self.subTest("valid model instance"):
+            self.assertTrue(TimelineLog.objects.for_object(self.article))
+
+        with self.subTest("invalid model instance"), self.assertRaises(AssertionError):
+            TimelineLog.objects.for_object("invalid argument")
```

Comment:
this is not semantically correct - if you do this, it would need to be a type error. Additionally, `assert` should really not be used for input validation, since they [get completely ignored](https://stackoverflow.com/questions/2055557/what-is-the-use-of-the-o-flag-for-running-python) when running python in "optimized" mode

They are mostly useful for type checking and "living documentation"/expression of intent.

A better way would be:

```py
with self.subTest("unsaved model instance"):
    qs = TimelineLog.objects.for_object(Article())
    
    self.assertIsInstance(qs, models.QuerySet)
    self.assertEqual(len(qs), 0)
```

* I don't like the `self.assertTrue` or `self.assertFalse` assertions on the queryset, since they rely on implicit conversion to bools
* Let's make it explicit that we always expect back a queryset, which may be empty.